docs-src/modules/developing/pages/tables.adoc

```diff
@@ -128,22 +146,20 @@ You can delete a table using its name:
 Python::
 +
 [source,python]
-.with Python:
 ----
 from kaskada import table
 from kaskada.api.session import LocalBuilder
 
 session = LocalBuilder().build()
 
-table.delete_table(table = "Purchase")
+table.delete_table("Purchase")
```

Comment:
Any reason for removing the argument name?

Reply:
Just trying to make things consistent.  the view & materialization docs weren't using argument names.